themes/cakephp/static/css/default.css

```diff
@@ -158,6 +165,9 @@
 
 
 /* -- Sidebar -- */
+.sidebar {
+    font-family: "Open Sans", sans-serif;
```

Comment:
Not sure we need yet another sans-serif typeface on the page. This will make it 3 in addition to Raleway, and Helvetica.